manila/share/drivers/netapp/dataontap/cluster_mode/lib_base.py

```diff
@@ -1969,6 +1969,12 @@ def revert_to_snapshot(self, context, snapshot, share_server=None):
                          self._get_backend_snapshot_name(snapshot['id']))
         LOG.debug('Restoring snapshot %s', snapshot_name)
         vserver_client.restore_snapshot(share_name, snapshot_name)
+        volume = vserver_client.get_volume(share_name)
+
+        # When calculating the size, round up to the next GB.
```

Comment:
We should better put this custom size calculation into an if-else-condition respecting the `netapp_volume_provision_net_capacity` option

Reply:
fixed